src/Storage/Azure.Storage.psd1

```diff
@@ -115,7 +115,9 @@ CmdletsToExport = 'Get-AzureStorageTable', 'New-AzureStorageTableSASToken',
                'Set-AzureStorageContainerStoredAccessPolicy', 
```

Comment:
Please update the changelog with these changes.

Reply:
change log is added